test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13265,33 +13245,21 @@ def create_pelog(self, body=None, src=None, filename=None):
                          ' creating ' + filename + ' with body\n' + '---')
         if body is not None:
             self.logger.info(body)
-            (fd, src) = self.du.mkstemp(prefix='pbs-pelog')
-            os.write(fd, body)
-            os.close(fd)
+            src = self.du.create_temp_file(prefix='pbs-pelog', body=body)
         elif src is not None:
             _b = open(src)
             self.logger.info("\n".join(_b.readlines()))
             _b.close()
         self.logger.info('---')
 
-        ret = self.du.run_copy(self.hostname, src, pelog, sudo=True)
+        ret = self.du.run_copy(self.hostname, src, pelog,
+                               preserve_permission=False, sudo=True)
         if body is not None:
             os.remove(src)
         if ret['rc'] != 0:
             self.logger.error('error creating pelog ')
             return False
 
-        ret = self.du.chown(self.hostname, path=pelog, uid=0, gid=0, sudo=True,
-                            logerr=False)
-        if not ret:
-            self.logger.error('error chowning pelog to root')
-            return False
-
-        ret = self.du.chmod(self.hostname, path=pelog, mode=0755, sudo=True)
```

Comment:
I believe we need this as this pelog script. Which must have executable permission.
Also please check for owner for that file also, as far as I remember it should be root only.